libbeat/common/schema/schema.go

```diff
@@ -57,6 +60,10 @@ func (conv Conv) Map(key string, event common.MapStr, data map[string]interface{
 			err.Optional = conv.Optional
 			err.Required = conv.Required
 		}
+		if conv.IgnoreAllErrors {
+			logp.Debug("libbeat.common", "ignoring error for key %q: %s", key, err)
```

Comment:
Is this debug selector copied from somewhere else? It would be nice if it included `schema` in it, for better context

Reply:
didn't copy it, it looks like the first log usage at the package.
I'm not sure about the name, are you suggesting `libbeat.common.schema`?